### ISSUE TITLE
153. Find Minimum in Rotated Sorted Array has a bad return value for the Java solution

### ISSUE THREAD

**Issue by lenocinor** [LeetCode-Feedback/LeetCode-Feedback#7445]
#### Your LeetCode username
ed_younskevicius

#### Category of the bug
- [ ] Question
- [*] Solution
- [ ] Language
- [ ] Missing Test Cases 

#### Description of the bug
-1 is a bad return value for the Java solution code for this function, because that is a valid value that may be returned for some user input. The final line of code should return a value outside the range -5000 to 5000 that problem states, like 5001 or Integer.MAX_VALUE or something like that.

I realize that in practice it doesn't matter, because this algorithm can never return this value. (It's coded in such a way that this return never happens.) But in real code, a bug may creep into this code or one of its dependencies, and then the error here may be very non-obvious when it is returned as -1. It's teaching programmers bad practice.

TL/DR: The last line of the Java solution should be

return 5001;

or

return Integer.MAX_VALUE;

#### Code you used for Submit/Run operation
N/A

#### Language used for code
N/A

#### Expected behavior
N/A

#### Screenshots
N/A

#### Additional context
N/A

**Comment by lenocinor** [comment]
PS: The problem description for "33. Search in Rotated Sorted Array" is flawed in the same way. We should not be returning -1 if the target is not found.

**Comment by LC-Pam** [comment]
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Pam** [comment]
Thank you for your time. We've submitted your feedback to update the problem. We appreciate your support!
